### DISCUSSION TITLE
Welcome to CommonTools Discussions!

### DISCUSSION THREAD

**Discussion by lgxlsm** [lgxlsm/CommonTools#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lgxlsm** [comment]
此项目用于收录，日常工作中常用代码

**Reply by maguaxiuer** [reply]
66666

**Reply by lgxlsm** [reply]
斌哥

**Comment by maguaxiuer** [comment]
I have a question

**Reply by lgxlsm** [reply]
回复一下问题

**Reply by lgxlsm** [reply]
再回复一次

**Reply by maguaxiuer** [reply]
1+1等于几?
